homeassistant/components/hunterdouglas_powerview/__init__.py

```diff
@@ -37,29 +35,23 @@
 
 async def async_setup_entry(hass: HomeAssistant, entry: PowerviewConfigEntry) -> bool:
     """Set up Hunter Douglas PowerView from a config entry."""
-
     config = entry.data
-
-    hub_address = config[CONF_HOST]
-    api_version = config.get(CONF_API_VERSION, None)
+    hub_address: str = config[CONF_HOST]
+    api_version: int | None = config.get(CONF_API_VERSION, None)
```

Comment:
```suggestion
    api_version: int | None = config.get(CONF_API_VERSION)
```

Reply:
Good catch.  Was so focused on the issue, I didn't see it 😃